src/parser.cpp

```diff
@@ -479,6 +479,103 @@ options_metadata parser::load_settings()
         "node.refresh_transactions",
         value<bool>(&configured.node.refresh_transactions),
         "Request transactions on each channel start, defaults to false."
+    )
+
+    /* [bitcoin] */
+    (
+        "bitcoin.retargeting_factor",
+        value<uint32_t>(&configured.bitcoin.retargeting_factor),
+        "The retargeting factor, defaults to 4."
+    )
+    (
+        "bitcoin.block_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.block_spacing_seconds),
+        "The target block spacing time in seconds, defaults to 600."
+    )
+    (
+        "bitcoin.timestamp_limit_seconds",
+        value<uint32_t>(&configured.bitcoin.timestamp_limit_seconds),
+        "The amount of seconds the timestamp is allowed to drift, defaults to 7200."
```

Comment:
"The future timestamp allowance in seconds, defaults to 7200."

Reply:
Done.